common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -75,12 +75,47 @@ public static Provider newProvider() {
 
     /**
      * Constructs a new {@link Provider} with the given name.
+     *
+     * @deprecated Use {@link #newProviderBuilder()} instead.
      */
+    @Deprecated
     public static Provider newProvider(String providerName) {
         checkAvailability();
         return new OpenSSLProvider(providerName);
```

Comment:
might be good to call the other constructor to be explicit about the value for trust manager.

Reply:
Done.